File: js-coverage/code.js
function shouldIDeployOn(dayOfWeek) {
    if (dayOfWeek == 'Friday') {
        return "How about Monday?";
    } else {
        return "Yes";
    }
}

function shouldIWatchThisMovie(movie) {
    if (movie == 'Joker') {
        return "What about Twilight...?";
    } else {
        return "Yes";
    }
}

module.exports = { shouldIDeployOn, shouldIWatchThisMovie };

File: clojure-coverage/code.clj
(defn should-i-deploy-on [day-of-week]
  (if (= day-of-week "Friday")
    "How about Monday?"
    "Yes"))

(defn should-i-watch-this-movie [movie]
  (if (= movie "Joker")
    "What about Twilight...?"
    "Yes"))
